Doc/library/decimal.rst

```diff
@@ -2159,12 +2159,12 @@ for medium-sized numbers and the `Number Theoretic Transform
 <https://en.wikipedia.org/wiki/Discrete_Fourier_transform_(general)#Number-theoretic_transform>`_
 for very large numbers.
 
-The context must be adapted for exact arbitrary precision arithmetic. :attr:`Emin`
-and :attr:`Emax` should always be set to the maximum values, :attr:`clamp`
-should always be 0 (the default).  Setting :attr:`prec` requires some care.
+The context must be adapted for exact arbitrary precision arithmetic. :attr:`!Emin`
+and :attr:`!Emax` should always be set to the maximum values, :attr:`!clamp`
+should always be 0 (the default).  Setting :attr:`!prec` requires some care.
```

Comment:
```suggestion
The context must be adapted for exact arbitrary precision arithmetic. :attr:`~Context.Emin`
and :attr:`~Context.Emax` should always be set to the maximum values, :attr:`~Context.clamp`
should always be 0 (the default).  Setting :attr:`~Context.prec` requires some care.
```

Likewise, these are all real attributes of `Context`, just not formally documented.